pkg/controller/sharedvolume/sharedvolume_controller.go

```diff
@@ -175,8 +178,11 @@ func (r *ReconcileSharedVolume) Reconcile(request reconcile.Request) (reconcile.
 		// an error path whose behavior we don't want to disrupt.
 		// Note that we don't clear Status.ClaimRef: if it's set, it might help track
 		// down the cause of the error.
-		r.markStatus(reqLogger, sharedVolume, awsefsv1alpha1.SharedVolumeFailed, err.Error())
-		return reconcile.Result{}, err
+		err := r.markStatus(reqLogger, sharedVolume, awsefsv1alpha1.SharedVolumeFailed, err.Error())
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		return reconcile.Result{}, nil
```

Comment:
This one I don't think we want to change -- see the code comment above. I think the linter is satisfied if you're explicit about ignoring the error return:
```suggestion
		_ := r.markStatus(reqLogger, sharedVolume, awsefsv1alpha1.SharedVolumeFailed, err.Error())
		return reconcile.Result{}, err
```